frontend/packages/dev-console/src/components/topology/operators/TopologyOperatorBackedResources.tsx

```diff
@@ -1,22 +1,153 @@
 import * as React from 'react';
+import { Link } from 'react-router-dom';
+import {
+  GroupVersionKind,
+  K8sResourceKind,
+  modelFor,
+  referenceFor,
+  referenceForGroupVersionKind,
+  referenceForModel,
+} from '@console/internal/module/k8s';
+import {
+  ClusterServiceVersionKind,
+  ClusterServiceVersionModel,
+  CRDDescription,
+  providedAPIsFor,
+  referenceForProvidedAPI,
+} from '@console/operator-lifecycle-manager/src';
+import {
+  flattenCsvResources,
+  linkForCsvResource,
+} from '@console/operator-lifecycle-manager/src/components/k8s-resource';
+import { Firehose, ResourceIcon, StatusBox } from '@console/internal/components/utils';
 import { TopologyDataObject } from '../topology-types';
 import TopologyGroupResourcesPanel from '../components/TopologyGroupResourcesPanel';
 
+type OperatorResourcesProps = {
+  namespace: string;
+  resources?: {
+    [kind: string]: { data: K8sResourceKind[] };
+  };
+  loaded?: boolean;
+  loadError?: string;
+  flatten: (resources: { [kind: string]: { data: K8sResourceKind[] } }) => K8sResourceKind[];
+  linkForResource?: (obj: K8sResourceKind) => React.ReactElement;
+};
+
+const OperatorResources: React.FC<OperatorResourcesProps> = ({
+  namespace,
+  resources,
+  loaded,
+  loadError,
+  flatten,
+  linkForResource,
+}) => {
+  const manifestResources = flatten(resources);
+  return (
+    <StatusBox data={manifestResources} loaded={loaded} loadError={loadError} label="Resources">
+      <TopologyGroupResourcesPanel
+        manifestResources={flatten(resources)}
```

Comment:
in line 45 https://github.com/openshift/console/pull/6591/files#diff-fd65de433ab9e684dee076b3cc686df8R45, we have it it in const , can we use that?